webpack.config.js

```diff
@@ -5,30 +5,43 @@ const srcDir = path.join(__dirname, 'client', 'src')
 
 const config = {
   context: srcDir,
-
+  
   entry: {
     index: './index.js'
   },
-
+  
   output: {
     path: path.join(__dirname, 'client', 'dist'),
     filename: 'bundle.js'
   },
-
+  
   module: {
-    rules: [{
-      test: /\.js$/,
-      exclude: /node_modules/,
-      use: {
-        loader: 'babel-loader'
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader'
+        }
+      }, {
+        test: /\.css$/,
+        loaders: [
+          'style-loader?sourceMap',
+          'css-loader?modules&importLoaders=1&localIdentName=[local]__[hash:base64:5]'
+        ]
+      },
+      {
+        test: /\.(png|jpg|gif)$/,
+        use: [
+          {
+            loader: 'url-loader',
+            options: {
+              limit: 8192
+            }
+          }
+        ]
```

Comment:
Since there is only one loader here, try using the syntax:
```js
use: {
  loader: 'url-loader',
  options: {
    limit: 8192
  }
}
```
Notice it is not in an array.